### PR TITLE
add 'react-native-ultimate-icons'

### PR DESCRIPTION
**Pull request by pcamarajr**
# 📝 Why & how

It adds the [react-native-ultimate-icons](https://www.npmjs.com/package/react-native-ultimate-icons) to the directory

# ✅ Checklist

- [X] Added library to **`react-native-libraries.json`**
- [ ] Updated library in **`react-native-libraries.json`**

- [ ] Documented in this PR how to use the feature or replicate the bug.
- [ ] Documented in this PR how you fixed or created the feature.

### CODE REVIEW

File: react-native-libraries.json [hunk 1]
```diff
@@ -9678,5 +9678,13 @@
     "android": true,
     "web": true,
     "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/pcamarajr/react-native-ultimate-icons",
+    "npmPkg": "react-native-ultimate-icons",
+    "examples": ["https://github.com/pcamarajr/react-native-ultimate-icons/tree/main/example/src"],
```
Comment (Simek) [inline react-native-libraries.json:9685]: Hello @pcamarajr, thank you for adding new library to the directory! 👍

Overall looks good, just a small nit related to data, before we can merge it in:
```suggestion
    "examples": ["https://github.com/pcamarajr/react-native-ultimate-icons/tree/main/example"],
```

Comment (pcamarajr) [reply react-native-libraries.json]: Got it. As the npm package name and girhub repository are the same, we can ignore the npmPkg value. Is that it it?

Other that, thank you as well for supporting the `directory`.; 🚀

Comment (Simek) [reply react-native-libraries.json]: Yup - https://github.com/react-native-community/directory#npmpkg - basically this field is dedicated for scoped packages.

**Review by Simek — APPROVED** [review]
Thank you for the updates, LGTM! 🚀

**Pull request merged**